tests/pytests/test_erisieve.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+import psi4
+import itertools
+
+pytestmark = pytest.mark.quick
+
+
+def test_no_screening_schwarz():
+    """Checks the number of shell quartets screened with Schwarz screening.
+    No shell quartets should be screened with a threshold of 0.0"""
+
+    Ne = psi4.geometry("""
+      Ne 0.0 0.0 0.0
+      Ne 4.0 0.0 0.0
+      Ne 8.0 0.0 0.0
+    """)
+
+    _, wfn = psi4.energy('hf/cc-pvdz', return_wfn=True)
+    basis = wfn.basisset()
+    sieve_schwarz = psi4.core.ERISieve(basis, 0.0, False)
+
+    shell_inds = range(basis.nshell())
+    quartets = itertools.product(shell_inds, shell_inds, shell_inds, shell_inds)
+
+    screen_count = 0
+    for m, n, r, s in quartets:
+        if not sieve_schwarz.shell_significant(m, n, r, s):
+            screen_count += 1
+
+    assert psi4.compare_values(0, screen_count, 'Quartets Schwarz Screened, Cutoff 0')  #TEST
```

Comment:
`tests/pytests/` is an odd case where you want to `assert compare_fn` directly instead of `psi4.compare_fn`. At the top, do `from .utils import compare`. `compare` (or `compare_integers`) are used when floating-point comparison not needed. The former is a new qcel fn and you can feed it integer arrays, too.

the reason for using compare functions from utils instead of psi4 is that the latter returns True/raise for pass/fail, whereas for `assert`, we want True/False.

Reply:
Thanks, that explanation makes a lot of sense!